main.js

```diff
@@ -1,74 +1,48 @@
 const form = document.getElementById('form');
-const titleInput = document.getElementById('title');
-const authorInput = document.getElementById('author');
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
 const booksList = document.getElementById('booksList');
 
 let books = [];
 let id = 0;
 
-//  Inner Html Generating Books
-
-function createBook(book) {
-  const newBook = document.createElement('li');
-  newBook.innerHTML = `
-    <div>${book.title}</div>
-    <div>${book.author}</div>
-    <button type="button" class="remove-btn" data-id="${book.id}">Remove</button>
-    <hr>
-  `;
-  booksList.appendChild(newBook);
-}
-// Clearing Fields
-
-function clearingFields() {
-  titleInput.value = '';
-  authorInput.value = '';
+function addBook() {
+  localStorage.setItem('books', JSON.stringify(books));
 }
```

Comment:
- [ ] Well done on implementing the local storage :+1: . Rather than using `function addBook()` outside the `class Collection{}` it would be better to use it inside the class same as `loadBooks(), createBook()` etc. methods as this project is mainly class-based. Kindly fix this issue.